udmis/udmis.iml

```diff
@@ -35,8 +35,10 @@
     <orderEntry type="library" name="Gradle: commons-cli:commons-cli:1.5.0" level="project" />
     <orderEntry type="library" name="Gradle: com.google.cloud:google-cloud-logging:3.14.9" level="project" />
     <orderEntry type="library" name="Gradle: io.github.clearblade:clearblade-cloud-iot:1.0.5" level="project" />
+    <orderEntry type="library" name="Gradle: io.etcd:jetcd-core:0.6.1" level="project" />
```

Comment:
Out of curiosity, is it intended that IntelliJ files are committed? In general I recommend excluding the IntellIJ configs, because they can be regenerated from Gradle, and it tends to clog up the repository.

It's not a huge problem, but it might be cleaner to deleted them.

Reply:
Yes -- I generally follow the IDEA guidelines on this... include everything except the workplace files. They've sequestered the "individual specific" stuff in one place. I've never seen it actually clog up the repository... (yes, it's redundant but mostly the tools don't care and the churn is pretty low".  If it's causing a problem somewhere let me know and we can address (sometimes personal paths creep their ways in), but ultimately it's generally useful for sharing some things like run/debug configurations etc...